app/views/find_local_council/district_and_county_council.html.erb

```diff
@@ -40,10 +47,19 @@
             local_authority_name: @county["name"]), @county["homepage_url"],
             class: "govuk-link",
             data: {
-              module: 'gem-track-click',
+              module: 'gem-track-click ga4-link-tracker',
               track_category: 'pageElementInteraction',
               track_action: 'countyLinkClicked',
-              track_label: @county["homepage_url"]
+              track_label: @county["homepage_url"],
+              ga4_link: {
+                event_name: "information_click",
```

Comment:
The implementation guide for this also has `'action':'information click',`. Should this be added?

Reply:
Added.